packages/frontend-shared/src/components/Checkbox.vue

```diff
@@ -38,19 +38,23 @@
 </template>
 
 <script lang="ts" setup>
+import { useModelWrapper } from '../composables'
+
 type InputState = 'success' | 'danger' | 'default'
 
-withDefaults(defineProps<{
+const props = withDefaults(defineProps<{
   id: string
-  modelValue: boolean
+  modelValue: any
```

Comment:
Why do we need to change the type here?

Reply:
Because in this case the value that I'm passing is a list of strings, not a boolean